### PR TITLE
FEAT: Added Input component

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-15 14-45-09

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+:root {
+    --success: #008000;
+    --warning: #FFA500;
+    --error: #FF0000;
+    --medium-font-size: 1.5rem;
+}
+
+.input-section {
+    margin-top: auto;
+    margin-bottom: auto;
```
Comment (yashpatel1797) [inline Components/Input/input.css]: try to use shorthand properties.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

File: Components/Input/input.css (hunk 1 above)
Comment (kotesh-arya) [reply Components/Input/input.css]: Sure, Noted that 👍🏻

File: Components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+.warning {
+    border: 2px solid var(--warning);
+
+}
+
+.error {
+    border: 2px solid var(--error);
+
+}
+.input-label-success{
```
Comment (zabihshaik) [inline Components/Input/input.css]: Better to use Prettier for Code Formatting

File: Components/Input/input.css (hunk 1 above)
Comment (zabihshaik) [reply Components/Input/input.css]: Similar suggestion as Yash

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a couple of comments. Plz check.

File: Components/Input/input.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/Input/input.css]: Done that 👍🏻

**Review by vinc3nati — COMMENTED** [review]
Good to proceed!

**Pull request merged**